data/hai/hai.txt

```diff
@@ -11,28 +11,50 @@
 # You should have received a copy of the GNU General Public License along with
 # this program. If not, see <https://www.gnu.org/licenses/>.
 
-shipyard "Hai Basics"
-	"Aphid"
-	"Lightning Bug"
+# For Hai shipyards their "Advanced" ships are restricted to the planets where the ships are actually manufactured.
+# In the case of "Intermediate" shipyards being required, those should be restricted to planets with substantial industrial bases.
+# The basics are those which can (but might not) be exported and sold pretty much anywhere that could provide reasonable maintenance services.
+
+# Imo Loo Mer Corp shipyards:
+shipyard "Imo Loo Mer Advanced"
```

Comment:
Small organizational nitpick, but the convention elsewhere is to define the basic shipyard first.